### ISSUE TITLE
Feature Request: Automatically reload the database when SQLPro becomes the active app

### ISSUE THREAD

**Issue by GaryMeyers1** [hankinsoft/SQLPro#829]
SQLPro has the "Reload database on external changes" option which I can understand that many users prefer to have turned off.  However, it would be convenient if there was an option to automatically reload the database when the user makes SQLPro the active app.  This would prepare the database with the latest values for the user to examine and manipulate.

**Comment by hankinsoft** [comment]
To confirm, you want it only to reload once the app becomes active?

**Comment by GaryMeyers1** [comment]
Yes, once every time that the user comes back to SQLPro. It's easy to forget to reload and then if you try to save the database you see the message about it can't save because there are external changes.

**Comment by kpconnell** [comment]
Yes off by default this would not be viable in large environments.
